### ISSUE TITLE
Typos for Chapter 5 and the appendices

### ISSUE THREAD

**Issue by benjub** [metamath/metamath-book#149]
Some typos for Chapter 5 and the appendices. The (page, line) references are made to metamath.pdf dated 2019-05-24.

153: if the file name contains slashes (/) --> if the file name contains slashes (/) or spaces [again on p. 158, thrice]
155: If you need to use a slash --> If you need to use a slash or a space
159: $a and $p and $[ and $] are not in typewriter font
159: will be inclfile argument : not clear
159, sec 5.4.3: Optional command qualifier --> Optional command qualifiers
160: show statement label --> show statement label-match (?) (several times, and have to modify accordingly the description)
160, sec 5.4.5: Add the /join qualifier and its description
161: show proof label --> show proof label-match (?)
161: start_column: what is the default ?
162: show usage label --> show usage label-match (?)
162, sec 5.5.3: show trace_back --> add "label" or "label-match" to the "Syntax" line 
162, sec 5.5.3: Counts --> Count (everywhere else, the infinitive form is used)
162, sec 5.5.4: "verify" begins a sentence; I suggest to write: The command "verify proof *"
163: add the command "verify markup" somewhere ?
166, sec 5.6.3: indicate that the default for empty_substitution is "off" ?
166, sec 5.6.5: isn't /essential the default now ?
167: 5.6.6: "[/no_unify]" does not appear on the "Syntax" line
170,-5 (towards end of sec 5.6.12): "/no_distinct" should begin a new line
173, 15: there is a very large "HTML" (in "HTML version of the document"); probably forgot smallcaps in the TeX
173: Currently there is no separate command to check for these errors: now, there is "verify markup", isn't it ?
144: htmlvarcolors --> htmlvarcolor (twice plus index)
173: the "Related Theorems" links --> "the Nearby theorems" links [no capital "T"]
175: file processing --> file-processing
175: parse --> isn't it rather "lexe" or "tokenize"? (to me, parsing implies a grammar)
177--179: Is it possible to have this as a LaTex table, with caption e.g. "Table 1: List of tokens used in set.mm" and \label{table:...}, to which one could refer using \ref{table:...}? Maybe it's too hard when it spans several pages.
187: must either contain no variables, or if they do, they may not have variables in common --> may not have variables in common [to me, the first part is redundant: if they do not contain variables, then they have none in common]
187: The set of axiomatic statements are a subset --> The set of axiomatic statements is a subset
196: We remove --> More explicitly, we remove [else, it may sound like one is doing twice the same thing]

**Comment by david-a-wheeler** [comment]
TODO:

> 160, sec 5.4.5: Add the /join qualifier and its description

**Comment by david-a-wheeler** [comment]
I got through this one:

> 163: add the command "verify markup" somewhere ?

I need to do other things for now, so I'll get back to the rest later...

**Comment by david-a-wheeler** [comment]
> 177--179: Is it possible to have this as a LaTex table, with caption e.g. "Table 1: List of tokens used in set.mm" and \label{table:...}, to which one could refer using \ref{table:...}? Maybe it's too hard when it spans several pages.

It's already a LaTeX table, specifically a longtabu table.  You're right that LaTeX often doesn't like multi-page tables, but longtabu is one of the few constructs that can handle it.  I had to do a lot of trial and error before I got this table to work, so I don't think we can get much better. I think referring to it through the Appendix label is adequate for purpose... there's really no advantage to a table caption when the entire appendix is this one table.
